### DISCUSSION TITLE
Escape does not work

### DISCUSSION THREAD

**Discussion by a-Rahmati** [JetBrains/ideavim#439] [Q&A]
Hello,

I use IdeaVim on PyCharm on Ubuntu.

I used to enter the command mode by pressing the escape key. I think PyCharm has updated IdeaVim and now escape does not work.

What happened to IdeaVim? Seems lots of other things (like Ctrl+V) changed.

Can anyone tell me how to enter command mode now?

**Comment by AlexPl292** [comment]
Hi, do you have IdeaVim enabled in the status icon?
![image](https://user-images.githubusercontent.com/4203721/148943773-428f37fa-ee68-4de6-aab7-5d7e98da8005.png)

Could you please share the log (https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)?
